cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/snowbridge_v2_outbound.rs

```diff
@@ -150,24 +153,16 @@ pub fn register_relay_token_from_asset_hub_user_origin() {
 
 		assert_ok!(
 			<AssetHubWestend as AssetHubWestendPallet>::SnowbridgeSystemFrontend::register_token(
-				RuntimeOrigin::signed(AssetHubWestendSender::get()),
-				bx!(VersionedLocation::from(Location { parents: 1, interior: [].into() })),
+				RuntimeOrigin::signed(AssetHubWestendAssetOwner::get()),
+				bx!(VersionedLocation::from(usdt_at_ah_westend())),
 				AssetMetadata {
-					name: "wnd".as_bytes().to_vec().try_into().unwrap(),
-					symbol: "wnd".as_bytes().to_vec().try_into().unwrap(),
-					decimals: 12,
-				},
+					name: "usdt".as_bytes().to_vec().try_into().unwrap(),
+					symbol: "usdt".as_bytes().to_vec().try_into().unwrap(),
+					decimals: 6,
+				}
 			)
 		);
 	});
-
-	BridgeHubWestend::execute_with(|| {
-		type RuntimeEvent = <BridgeHubWestend as Chain>::RuntimeEvent;
-		assert_expected_events!(
-			BridgeHubWestend,
-			vec![RuntimeEvent::EthereumOutboundQueueV2(snowbridge_pallet_outbound_queue_v2::Event::MessageQueued{ .. }) => {},]
-		);
-	});
```

Comment:
Why remove this check? I know this test main focus is whether `AssetHubWestend::SnowbridgeSystemFrontend::register_token()` is successful, but it's still nice to assert it goes through all the way :)

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/57/commits/cc1355f9f96424a8b88f5ab9d61a86dd0c7d68e6